spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureConfigPropertySource.java

```diff
@@ -43,12 +59,40 @@ public Object getProperty(String name) {
 
     public void initProperties() {
         // * for wildcard match
+        // Reading in Configurations
         QueryOptions queryOptions = new QueryOptions().withKeyNames(context + "*").withLabels(label);
         List<KeyValueItem> items = source.getKeys(storeName, queryOptions);
-
         for (KeyValueItem item : items) {
             String key = item.getKey().trim().substring(context.length()).replace('/', '.');
             properties.put(key, item.getValue());
         }
+
+        // Reading In Features
+        List<String> queries = new ArrayList<String>();
+        queries.add("*appconfig*");
+        queryOptions = new QueryOptions().withKeyNames(queries)
+                .withLabels(label);
+        items = source.getKeys(storeName, queryOptions);
+        FeatureSet featureFile = new FeatureSet();
```

Comment:
nit: Is this variable name `featureFile` on purpose? Do not understand why the `File` here. Or should be a `featureSet` ?